sufia-models/app/models/concerns/sufia/works/generic_work.rb

```diff
@@ -5,7 +5,8 @@ module GenericWork
 
     include Sufia::Works::Work
     include Sufia::Works::CurationConcern::WithBasicMetadata
+    include Sufia::Works::GenericWork::Metadata
     include Sufia::GenericFile::Permissions
     include Sufia::GenericFile::Metadata
```

Comment:
@yinlinchen I think we want to only use the work metadata and remove the file metadata from the basic work structure.  Later when when index the work we can then include the file metadata.  Does that make sense?  For this PR I would just remove this line.